components/side-bar.tsx

```diff
@@ -3,76 +3,171 @@
 import Image from "next/image";
 import SideDrawer from "./side-drawer";
 import Link from "next/link";
-import { FC } from "react";
+import { FC, useState } from "react";
 import { usePathname } from "next/navigation";
 import { routes } from "@/routes";
 import { SheetClose } from "./ui/sheet";
 import { ROUTES } from "@/utils/constants";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+} from "@/components/ui/select";
+// This will be later used when the organizations are set on the backend
+// import { useSelector } from "react-redux";
+// import { RootState } from "@/types";
 
 interface SideBarProps {
   inSideDrawer?: boolean;
 }
 
+interface Organization {
+  id: string;
+  label: string;
+  image: string;
+}
+
 const SideBar: FC<SideBarProps> = ({ inSideDrawer = false }) => {
   const pathname = usePathname();
 
+  const [isHovered, setIsHovered] = useState<number>();
```

Comment:
you don't need to create a state to set hover effect, instead you can use `group-hover` from tailwind css